enterprise-shield/AMQP/rabbitmq/docker-compose.yml

```diff
@@ -5,7 +5,11 @@ services:
     ports:
       - "8000:8000"
     hostname: "kaazing.example.com"
+    environment:
+      GATEWAY_OPTS: -Dfeature.http.proxy=true -Djava.naming.provider.url=FILE://kaazing-gateway/bin"
   internal.example.com: 
     build: internal
+    environment:
+      GATEWAY_OPTS: -Dfeature.http.proxy=true -Djava.naming.provider.url=FILE://kaazing-gateway/bin"
```

Comment:
Why is the java.naming.provdier.url?  I wouldn't expect the .bindings file to be in bin.  And I don't think this would be needed for amqp service